src/vm/arm64/stubs.cpp

```diff
@@ -572,9 +572,27 @@ void FixupPrecode::Fixup(DataImage *image, MethodDesc * pMD)
 }
 #endif // FEATURE_NATIVE_IMAGE_GENERATION
 
+
 void ThisPtrRetBufPrecode::Init(MethodDesc* pMD, LoaderAllocator *pLoaderAllocator)
 {
-    _ASSERTE(!"ARM64:NYI");
+    WRAPPER_NO_CONTRACT;
+
+    int n = 0;
+    //Initially
+    //x0 -This ptr
+    //x1 -ReturnBuffer
+    m_rgCode[n++] = 0x91000010; // mov x16, x0
```

Comment:
Is this correct mov(register) (section C6.6.125 in arm64 manual) starts with encoding 0xa.......
which encoding is this? section(?)